### DISCUSSION TITLE
Welcome to MernBoilerPlate Discussions!

### DISCUSSION THREAD

**Discussion by GauravWalia19** [GauravWalia19/MernBoilerPlate#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by suryaprakash0010** [comment]
Hi , i am a full- stack developer looking to streamline my workflow using MERN boilerplate for a new project . i am excited to dig into codes , share some feedback , and connect with other builders.
